src/controllers/events/move-input.js

```diff
@@ -43,13 +43,11 @@ export default class MoveInput {
     }
     if (eventType === LEAVE_EVENT_TYPE) {
       this.enableLeaveEvent = enabled;
```

Comment:
Not related to this PR: 

I'm starting to think that we should maybe use a map with keys for the enabling of events, instead of creating duplicate `enable...` variables for all the events and gestures. 

But maybe the mapping is not one-to-one?